tasks/hacbs-java.yaml

```diff
@@ -0,0 +1,269 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
```

Comment:
@Michkov @stuartwdouglas - given the simularity with (but not exact copy of) https://github.com/redhat-appstudio/jvm-build-service/blob/main/deploy/base/maven-v0.2.yaml
does it make sense for this file to live and be maintained in the jvm-build-service repo?

Or maybe the common chunks (like the bash) could be broken out into parameters and then have the common code shared and referenced in the jvm-build-service repo?

Maybe the answer to all ^^ is "no", but there is some "non-trivial" duplication going on here.  It would be great if we could "share" it if that is feasible.

WDYT?

Reply:
In theory I am ok with it living here, it just needs to be documented in the jvm-build-service repo, and we need to make sure we have a strategy for testing any changes locally to allow for a local development workflow (which should not be a problem). 

My only real concern would be that co-ordinating changes across multiple repos can really start to hamper development velocity, but I think in this case it might not be that big a deal.

Reply:
This is one way how to use the jvm-build-service. Probably it could be rather named `hacbs-java-maven.yaml`, we should be able easily create `hacbs-java-s2i.yaml`. JVM build service can be used anytime when we set sidecar and maven mirror.

We can decide later how to de-duplicate.